### ISSUE TITLE
bullets in itemized text objects and colors

### ISSUE THREAD

**Issue by pierreganty** [otfried/ipe#161]
When using itemized or item text object, the color of the item indicator ("the bullets") remains black even though the text object has a different color.
A workaround goes as follows:
```
\item[\textcolor{<color-you-want>}{<item-you-want>}] blabla 
```

**Comment by otfried** [comment]
You can also fix the color once and for all:  http://ipe.otfried.org/manual/manual_33.html

But I still need to figure out why the bullets stay black...

**Comment by otfried** [comment]
I finally figured out why this happens.  It's fixed in 7.2.11.

**Comment by pierreganty** [comment]
Great! Thank you. Looking forward to 7.2.11
